library/src/main/res/layout/month.xml

```diff
@@ -6,15 +6,6 @@
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
     >
-  <TextView
```

Comment:
let's add a comment here explaining that the title view will be added programmatically